### ISSUE TITLE
.mothur files not writing

### ISSUE THREAD

**Issue by joverbey3** [blindner6/CEE6720_BIO6720#5]
Hi on step04, my code is as follows:

sbatch --export input=01_trimmed_reads/2012-summer,output=04_taxonomic profiles/2012-summer step04_vsearch.sbatch

And it is working for one of them, but it isn't working for the rest. The files are in the directory 04_taxonomic_profiles, but the mothur files aren't being written. Also, I know that 0's mean no matches, but it looks like all of them are 0's. I think I might be doing them incorrectly somehow, but I am not sure what. Thank you so much. 

![image](https://github.com/blindner6/CEE6720_BIO6720/assets/161337686/aa5b6d48-42c1-4388-8dee-e647ac5e374c)
![image](https://github.com/blindner6/CEE6720_BIO6720/assets/161337686/2ad84aa2-82a4-47d1-84bf-a9e916e6e5b7)
![image](https://github.com/blindner6/CEE6720_BIO6720/assets/161337686/b447c98c-41d2-4d76-bb80-9e6352dde384)

**Comment by bglindner** [comment]
Hey Jorja,

Can you show me just the bottom of the file? That would be easiest with `tail -n 5 mothur_file`

**Comment by joverbey3** [comment]
[A black background with white and blue lines  Description automatically generated]
Hey blake! It’s still a fat chunk of text but it generally looks like this

**Comment by bglindner** [comment]
Hmmm, that's not coming through via email reply. Can you try sharing that directly through the GitHub interface?

**Comment by joverbey3** [comment]
![image](https://github.com/blindner6/CEE6720_BIO6720/assets/161337686/9ed65a21-4066-4671-a8b7-968a2e5bcec2)
So sorry about the delay. I had a class at 3:30. I know there are supposed to be columns with 0s but I don't think all of them should have 0's

**Comment by bglindner** [comment]
Hey Jorja,

It's okay if there are a lot of zeros. As long as literally everything isn't a 0. For perspective, we have like a bajillion OTUs in the database but only ~3000 are detected in our dataset. You should take this file into R or Python and remove columns that sum to 0 and then take a look.

**Comment by bglindner** [comment]
You can also take a look at advice in issue #3! I shared some code that I would use to get these tables into an easily plot-able format.

**Comment by joverbey3** [comment]
Okay thank you so much! I was just concerned that I had too many 0's in comparison to non 0 columns. I really appreciate the help!

**Comment by bglindner** [comment]
Of course! Feel free to reply here or open another issue if anything else goes wrong with the `vsearch` results
